src/components/Layout/SideBar/SideBar.tsx

```diff
@@ -1,9 +1,216 @@
-import { Box } from "@mui/material";
+import { Avatar, Box, Divider, IconButton, Tooltip, Typography } from "@mui/material";
+import { ReactElement, useState } from "react";
+import DashboardIcon from '@mui/icons-material/Dashboard'
+import AccountCircleIcon from '@mui/icons-material/AccountCircle'
+import HandshakeIcon from '@mui/icons-material/Handshake'
+import LocalMallIcon from '@mui/icons-material/LocalMall'
+import TimelineIcon from '@mui/icons-material/Timeline'
+import TextSnippetIcon from '@mui/icons-material/TextSnippet'
+import TextsmsIcon from '@mui/icons-material/Textsms'
+import FullLogoOP from '../../../asset/logoOnlinePreps.png'
+import ShortLogoOP from '../../../asset/logo.PNG'
+import ChevronLeftIcon from '@mui/icons-material/ChevronLeft';
+import ChevronRightIcon from '@mui/icons-material/ChevronRight';
+import { theme } from "../../../utils/style/theme";
+import NavBar from "./NavBar";
+import Profile from "./Profile";
+
+export interface sideBarItem {
+    label: string
+    icon: ReactElement
+    link: string
+}
+
+export interface sideBarSection {
+    title: string
+    sideBarItems: sideBarItem[]
+}
 
 export default function SideBar() {
+    const [isSideOpen, setIsSideBarOpen] = useState<boolean>(true)
+    const sideBarSection: sideBarSection[] = [
```

Comment:
move this data structure to navBarSection function component so that this component will look more little.